<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>701</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font: 75 30pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 0);
font-weight:bold;

 
 </string>
    </property>
    <property name="text">
     <string>钞 纸 人 工 选 纸 条 码 扫 描 系 统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxCount_bad">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>601</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>3</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">
font:  22pt &quot;微软雅黑&quot;;
font-weight:bold;
color: rgb(255, 85, 0);</string>
    </property>
    <property name="title">
     <string>坏纸仓计数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLCDNumber" name="lcdCount_bad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>500</width>
       <height>350</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
border:none;</string>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0">
      <double>100.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>100</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>411</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  16pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>坏纸条码：</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_bad">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>70</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(251, 251, 251);
background-color: rgb(255, 255, 255);
font: 25pt &quot;Arial Black&quot;;
 
border:none;
color: rgb(0, 0, 0);
text-align: center;
font-weight:bold;
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>0000000000000000</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>410</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font:  16pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>令数</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_badPack">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(0,0,0);
font: 80pt &quot;黑体&quot;;
color: rgb(249, 83, 0);

</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>00</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBoxCount_good">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>731</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>3</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);
font:  22pt &quot;微软雅黑&quot;;
font-weight:bold;</string>
    </property>
    <property name="title">
     <string>好纸仓计数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLCDNumber" name="lcdCount_good">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>600</width>
       <height>371</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 250, 0);
border:none;

</string>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0">
      <double>500.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>500</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>461</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font:  16pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>好纸条码：</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_good">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>421</width>
        <height>70</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);
font: 25pt &quot;Arial Black&quot;;
 
border:none;
color: rgb(0, 0, 0);
text-align: center;
font-weight:bold;
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>0000000000000000</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gBoxLingNum_good">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>170</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font:  16pt &quot;微软雅黑&quot;;
</string>
     </property>
     <property name="title">
      <string>令数</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_goodPack">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: rgb(0, 255, 0);

font:90pt &quot;黑体&quot;;

background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>00</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Agency FB&quot;;
color: rgb(255, 255, 0);
font-weight;bold;
</string>
    </property>
    <property name="text">
     <string>操作人员：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_operator">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>8</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff;
font: 30pt &quot;Agency FB&quot;;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>张三</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangePaper">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>90</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;微软雅黑 light&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0.846591, x2:0.949, y2:0.0568182, stop:0 rgba(100, 100, 100, 255), stop:0.994318 rgba(243, 243, 243, 255));

color: rgb(255, 255, 255);
font-weight:bold;
 
</string>
    </property>
    <property name="text">
     <string>换 纸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetup">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>90</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>4</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0, y1:0.846591, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.994318 rgba(243, 243, 243, 255));</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>1321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid #fff;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>gBoxCount_bad</zorder>
   <zorder>label_15</zorder>
   <zorder>lb_operator</zorder>
   <zorder>btnChangePaper</zorder>
   <zorder>btnSetup</zorder>
   <zorder>line</zorder>
   <zorder>gBoxCount_good</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
